tests/robot-tests/tests/libs/public-common.robot

```diff
@@ -60,3 +60,10 @@ user checks methodology note
     [Arguments]    ${number}    ${displayDate}    ${content}
     user waits until element contains    css:#methodologyNotes li:nth-of-type(${number}) time    ${displayDate}
     user waits until element contains    css:#methodologyNotes li:nth-of-type(${number}) p    ${content}
+
+# Next.js will have returned a stale cached page for the last request and will have created a new cached page in
+# the background in the meantime, so request the page again.
+
+user reloads the stale cached page
+    sleep    3
```

Comment:
Why this particular sleep? Should this be based on `WAIT_CACHE_EXPIRY` or some other variable?

Reply:
No, it's arbitrarily the amount of time that Next needs to regenerate a page in the background.  Until this succeeds, the stale page will continue to be served up.  Could use WAIT_SMALL instead?

Reply:
`WAIT_SMALL` is 45 seconds by default though? That seems way too long for an explicit sleep.

Maybe it needs another env variable like `WAIT_NEXT_CACHE_EXPIRY` to set this more exactly for your environment.

Reply:
Done - WAIT_NEXT_ISR, as discussed ta